internal/v1/handler_blueprints.go

```diff
@@ -59,12 +66,31 @@ func (h *Handlers) CreateBlueprint(ctx echo.Context) error {
 		return err
 	}
 
+	if !blueprintNameRegex.MatchString(blueprintRequest.Name) {
+		return ctx.JSON(http.StatusUnprocessableEntity, HTTPErrorList{
+			Errors: []HTTPError{{
+				Title:  "Invalid blueprint name",
+				Detail: "The blueprint name must start and end with an alphanumeric character and may contain alphanumeric characters, underscores, hyphens, and spaces.",
```

Comment:
This does not quite match the regex, right? It can start/end with the space, we just need some characters that are not whitespace.

Reply:
ah, correct, thanks!